source/1_engine/rex_engine/include/rex_engine/filesystem/mount_roots.h

```diff
@@ -7,7 +7,7 @@ namespace rex
 {
   enum class MountRoot
```

Comment:
I think we can come up with a better name for this.
MountingPoint?

Reply:
What's the difference? It's the root path of a mount

Reply:
If it does not make any difference to you we can change this to MountingPoint
I find MountRoot a bit weird.
MountingPoint represents more as a location ( cuz it has point in the name ) in regards to MountRoot.